src/ncp/spinel.h

```diff
@@ -683,6 +688,8 @@ typedef enum
     SPINEL_PROP_NET_KEY_SWITCH_GUARDTIME
                                         = SPINEL_PROP_NET__BEGIN + 10, ///< [L]
 
+    SPINEL_PROP_NET_PSKC                = SPINEL_PROP_NET__BEGIN + 11, ///< [D]
```

Comment:
added missing properties to the source. Is there any doc on how to generate the final .html and .txt file?